wls-ergebnismeldung-service/src/main/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/service/begruendung/BegruendungService.java

```diff
@@ -0,0 +1,61 @@
+package de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.begruendung;
+
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.Begruendung;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.BegruendungRepository;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.BezirkUndWahlIDStapelart;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.util.ExceptionFactory;
+import jakarta.validation.constraints.NotNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.parameters.P;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class BegruendungService {
+
+    private final BegruendungRepository begruendungRepository;
+
+    private final BegruendungModelMapper begruendungModelMapper;
+
+    private final BegruendungValidator begruendungValidator;
+
+    private final ExceptionFactory exceptionFactory;
+
+    @PreAuthorize(
+        "hasAuthority('Ergebnismeldung_BUSINESSACTION_GetBegruendung')"
+                + " and @bezirkIdPermisionEvaluator.tokenUserBezirkIdMatches(#param.wahlbezirkID(), authentication)"
+    )
+    public BegruendungModel getBegruendung(@P("param") @NotNull final BegruendungReference begruendungReference) {
+        log.info("#getBegruendung");
+        begruendungValidator.validReferenceOrThrow(begruendungReference);
+
+        BezirkUndWahlIDStapelart id = begruendungModelMapper.toEmbeddedId(begruendungReference);
+        val begruendungFromRepo = getOrNull(id);
+        return begruendungFromRepo == null ? null : begruendungModelMapper.toModel(begruendungFromRepo);
+    }
+
+    @PreAuthorize(
+        "hasAuthority('Ergebnismeldung_BUSINESSACTION_PostBegruendung')"
+                + " and @bezirkIdPermisionEvaluator.tokenUserBezirkIdMatches(#param.wahlbezirkID(), authentication)"
+    )
+    public void setBegruendung(@P("param") @NotNull BegruendungModel begruendungToAdd) {
```

Comment:
bitte bei `postBegruendung` bleiben, wie im alten Code. entsprechend müssen dann auch die nested classes in den tests umbenannt werden
```suggestion
    public void postBegruendung(@P("param") @NotNull BegruendungModel begruendungToAdd) {
```